ui/ui-app/src/main/resources/static/js/ops-mgr/service-health/service-component-detail.html

```diff
@@ -94,7 +94,7 @@
                         <td md-cell>
                             {{alert.message}}
                         </td>
-                        <td md-cell hide show-gt-md >{{alert.latestTimestamp | date : "MM/dd/yyyy 'at' h:mm"}}</td>
+                        <td md-cell hide show-gt-md >{{'service-component-detail.time-locale' | translate:{date:alert.latestTimestamp} }} {{"service-component-detail.at" | translate}} {{'service-component-detail.time-locale2' | translate:{date:alert.latestTimestamp} }}</td>
```

Comment:
There are no English translations for `service-component-detail.at` and `service-component-detail.time-locale2`. Please fix this to only use `service-component-detail.time-locale`.